pkg/node/volumemgr.go

```diff
@@ -655,7 +655,9 @@ func (m *VolumeManager) discoverVolumeCRs(freeDrives []*drivecrd.Drive) error {
 				ll.Warnf("There is no part UUID for partition from device %v, UUID has been generated %s", bdev, partUUID)
 			}
 
-			volumeCR := m.k8sClient.ConstructVolumeCR(partUUID, api.Volume{
```

Comment:
For each volume we need to generate unique identifier and use it as a key. But this identifier shouldn't match to partition UUID since they might be the same on different nodes an devices. So we need to have name as autogenerated and Id in Spec to match partition UUID:
```kubectl describe volume 21d34bca-ab56-4f1d-a7fc-3deaab7742bc
Name:         21d34bca-ab56-4f1d-a7fc-3deaab7742bc <= autogenerated
Namespace:    
Labels:       <none>
Annotations:  <none>
API Version:  baremetal-csi.dellemc.com/v1
Kind:         Volume
Metadata:
  Creation Timestamp:  2020-11-09T13:59:23Z
  Finalizers:
    dell.emc.csi/volume-cleanup
  Generation:  1
  Managed Fields:
    API Version:  baremetal-csi.dellemc.com/v1
    Fields Type:  FieldsV1
    fieldsV1:
      f:metadata:
        f:finalizers:
          .:
          v:"dell.emc.csi/volume-cleanup":
      f:spec:
        .:
        f:Health:
        f:Id:
        f:Location:
        f:LocationType:
        f:Mode:
        f:NodeId:
        f:Size:
        f:Type:
    Manager:         node
    Operation:       Update
    Time:            2020-11-09T13:59:23Z
  Resource Version:  2742947
  Self Link:         /apis/baremetal-csi.dellemc.com/v1/volumes/21d34bca-ab56-4f1d-a7fc-3deaab7742bc
  UID:               5826a25c-ba2c-4fe3-b3d1-91b89274fd92
Spec:
  Health:         GOOD
  Id:             21d34bca-ab56-4f1d-a7fc-3deaab7742bc <= optional, should match to partition UUID. But can be empty in block(raw) mode
  Location:       71e303bf-0cf4-4e6d-927b-b0d7c6c61fe5
  Location Type:  DRIVE
  Mode:           FS
  Node Id:        f7f96e41-f7f9-43f9-8597-d8c48d05c82c
  Size:           8001563222016
  Type:           LVM2_member
Events:           <none>
```

Reply:
@rususlasan mentioned why we should use partUUID in Id https://github.com/dell/csi-baremetal/pull/152#discussion_r518681759

Reply:
We use partUUID for Id and autogenerated name for key now, I fixed it according to this discussion https://github.com/dell/csi-baremetal/pull/152#discussion_r518681759